scripts/patches.sh

```diff
@@ -0,0 +1,34 @@
+set -x -e
+
+# Copy old version of filters
+git checkout old-platforms -- old_platforms
+
+FOLDER_WITH_NEW_FILTERS="platforms"
+FOLDER_WITH_OLD_FILTERS="old_platforms"
+
+# Make diff-builder executable
+DIFF_BUILDER=node_modules/@adguard/diff-builder/dist/diff-builder
```

Comment:
diff-builder executable should be not taken from the dist
it either should be installed globally
or used like this
`npx diff-builder`